cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -38,6 +38,7 @@ func main() {
 	outPrefix := flag.String("outPrefix", "", "Output segments' prefix (no segments are generated by default)")
 	detectionFreq := flag.Int("detectionFreq", 0, "Run content-detection on every nth segment. No detection occurs for default frequency of 0.")
 	detectionSampleRate := flag.Uint("detectionSampleRate", 1, "Run content-detection on every nth frame of a particular segment, if detectionFreq > 0.")
+	concurrentSessionDelay := flag.Uint("concurrentSessionDelay", 300, "Delay before starting a new concurrent session (in ms)")
```

Comment:
What do you think about using [flag.Duration](https://pkg.go.dev/flag#Duration) here which is typically used for time based arguments which would also allow us to skip the extra conversion step on L223? The value provided would be of the form 300ms, 1.5h, 2h45m, etc. (basically anything accepted by [time.ParseDuration](https://pkg.go.dev/time#ParseDuration)).

Reply:
Neat, fixed!